minidump-processor/src/process_state.rs

```diff
@@ -362,7 +375,27 @@ impl CallStack {
                     write!(f, " + {:#x}", addr - module.base_address())?;
                 }
             } else {
-                writeln!(f, "{:#x}", addr)?;
+                write!(f, "{:#x}", addr)?;
+
+                // List off overlapping unloaded modules.
+
+                // First we need to collect them up by name so that we can print
+                // all the overlaps from one module together and dedupe them.
+
+                for (name, offsets) in &frame.unloaded_modules {
+                    write!(f, " (unloaded {}@", name)?;
+                    let mut first = true;
+                    for offset in offsets {
+                        if first {
+                            write!(f, "0x{:#x}", offset)?;
+                        } else {
+                            // `|` is our separator for multiple entries
+                            write!(f, "|0x{:#x}", offset)?;
+                        }
+                        first = false;
+                    }
```

Comment:
nit: You could use `map()` and `join()` and then write!() the whole string in one go (I'm assuming that `map()` does things in the order you'd expect).

Reply:
Ah yeah, that's the API I was trying to remember. I think I'm fine leaving it like this for now.